### PR TITLE
Remove `min_utxo_value + lovelace_per_utxo` check from minting to foreign wallet test

### PR DESCRIPTION
**Pull request by piotr-iohk**
- [x] Remove `min_utxo_value + lovelace_per_utxo` check from minting to foreign wallet test

### Comments

This test failed after #3368 was merged. Since the min UTxO value is a function of the size of the TxOut, rather than a constant value it is rather unpredictable to calculate this by hand (outside of wallet) especially for the wallets on the testnet which utxo may change between the runs. Removing this check then.

### Issue Number

n/a

### CODE REVIEW

File: test/e2e/spec/e2e_spec.rb [hunk 1]
```diff
@@ -1365,21 +1364,8 @@ def get_policy_id_from_decode(tx_decoded_mint_or_burn)
...
         wait_for_tx_in_ledger(@wid, tx_id)
-        src_after_minting = get_shelley_balances(@wid)
         target_after_minting = get_shelley_balances(@target_id)
 
-        # verify ADA balance is correct on src wallet:
-        # we are minting and sending to external address
-        # therefore the cost is fee + mintUTxOValue of ADA that is required
-        # for transfering the assets over the network
-        # in this the total `cost` is pure ADA minUTxOValue + 11 'utxo words' + fee
-        min_utxo_value = NETWORK.parameters['minimum_utxo_value']['quantity'].to_i
```
Comment (jonathanknowles) [inline test/e2e/spec/e2e_spec.rb:1376]: Hi @piotr-iohk 

Just wondering: is it possible to hard-code the expected minimum UTxO value(s) in the test?

This might give us a sanity check that the values are reasonable. 🤔 

If the minimum UTxO value does change in any way (in future), then:
- it would cause this test to fail;
- we would have a chance to see the new value;
- we would have a chance to verify that it looks reasonable (i.e., to repeat the sanity check).

Alternatively, if it's too difficult, then I think deleting this is fair enough.

Comment (piotr-iohk) [reply test/e2e/spec/e2e_spec.rb:1376]: Hm, actually at this moment is passes with this (i.e. with what GET network/parameters shows)... it failed last night, so I assume that it was due to the fact that there was different number/size of TxOut in the transaction than usual (since the utxo was probably differently shaped). The wallet is out there on the testnet and it runs through ~200 tests every night, it is rather hard to keep the utxo under control.

So in other words, if minUtxOValue depends on the number of outputs in a transaction it is rather hard to come up with a value to hardcode it. (At least I don't have idea how).

**Review by jonathanknowles — APPROVED** [review]
